### PR TITLE
conf.kde issue resolve 1

### PR DESCRIPTION
**Pull request by jamesthechamp**
1) change pattern just a line
2)few issue solved

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -49,7 +49,7 @@
   <div class="container center">
     <h1>Conf.kde.in 2016</h1>
     <span id="dates">March 4th-6th</span><br />
-    <a href="#"><span id="register">Register Now</span></a>
+    <a class="registerx" href="#registerx"><span id="register">Register Now</span></a>
```
Comment (shivamdixit) [inline index.html]: *registerx* is not a good name for a class. Write self descriptive names.

File: index.html [hunk 2]
```diff
@@ -69,11 +69,10 @@ <h2 id="schedulex" style="text-align:center;"><i class="material-icons custom_si
 <h5 class="center">Schedule Coming Soon</h5>
 <br /><br />
 </div>
-   <div id="just-a-line"></div>
-
-
 <section id="cfpx" class="module content">
-  <div class="container">
+<br /><br />
```
Comment (shivamdixit) [inline index.html]: Not indented properly.

File: index.html [hunk 3]
```diff
@@ -69,11 +69,10 @@ <h2 id="schedulex" style="text-align:center;"><i class="material-icons custom_si
...
 <br /><br />
 </div>
-   <div id="just-a-line"></div>
-
-
 <section id="cfpx" class="module content">
-  <div class="container">
+<br /><br />
+   <div id="just-a-line"></div>  
+<div class="container">
```
Comment (shivamdixit) [inline index.html]: Indent

File: index.html [hunk 4]
```diff
@@ -69,11 +69,10 @@ <h2 id="schedulex" style="text-align:center;"><i class="material-icons custom_si
 <h5 class="center">Schedule Coming Soon</h5>
 <br /><br />
 </div>
-   <div id="just-a-line"></div>
-
-
 <section id="cfpx" class="module content">
-  <div class="container">
+<br /><br />
+   <div id="just-a-line"></div>
```
Comment (shivamdixit) [inline index.html]: *just-a-line* is not a good id name

File: index.html [hunk 5]
```diff
@@ -82,17 +81,16 @@ <h2 class="center"><i class="material-icons custom_size">assignment</i>Call For
 We are also working on providing travel and accommodation assistance for speakers. Please leave a comment in the CFP form if you'd like to get assistance for the same. </p>
   </div>
 </section>
-
-<br /><br />
-   <div id="just-a-line"></div>
-
-
+<section id="speakerx" class="module content">
+  <br /><br />
```
Comment (shivamdixit) [inline index.html]: Indent.

File: index.html [hunk 6]
```diff
@@ -82,17 +81,16 @@ <h2 class="center"><i class="material-icons custom_size">assignment</i>Call For
...
   </div>
 </section>
-
-<br /><br />
-   <div id="just-a-line"></div>
-
-
+<section id="speakerx" class="module content">
+  <br /><br />
+   <div id="just-a-line"></div>
```
Comment (shivamdixit) [inline index.html]: Duplicate id name. Ids should always be unique.

File: index.html [hunk 7]
```diff
@@ -69,11 +69,10 @@ <h2 id="schedulex" style="text-align:center;"><i class="material-icons custom_si
...
 </div>
-   <div id="just-a-line"></div>
-
-
 <section id="cfpx" class="module content">
-  <div class="container">
+<br /><br />
+   <div id="just-a-line"></div>  
+<div class="container">
     <h2 class="center"><i class="material-icons custom_size">assignment</i>Call For Paper</h2>
```
Comment (shivamdixit) [inline index.html]: Use consistent class naming convention. When you have used "-" everywhere, then you should not use "_"

File: index.html [hunk 8]
```diff
@@ -82,17 +81,16 @@ <h2 class="center"><i class="material-icons custom_size">assignment</i>Call For
...
-
+<section id="speakerx" class="module content">
+  <br /><br />
+   <div id="just-a-line"></div> 
+   <div class="container">
       <h2 id="speakerx" style="text-align:center;"><i class="material-icons custom_size">perm_identity</i>Speakers</h2>
- 
-
+</section>
+  <!--
```
Comment (shivamdixit) [inline index.html]: Unnecessary code should not be present

File: index.html [hunk 9]
```diff
@@ -187,24 +185,28 @@ <h2 id="speakerx" style="text-align:center;"><i class="material-icons custom_siz
...
 <section id="registerx" class="module content">
+	
+  <br /><br />
+   <div id="just-a-line"></div>
   <div class="container regx center">
-    <h2 class="center"><i class="material-icons custom_size">done</i></h2>
+
+    <h2 class="center"><i class="material-icons custom_size">done</i>Join us</h2>
     <div class="register-btns">
     <a class="waves-effect waves-light btn-large" style="background-color:#2196F3;"><i class="material-icons right">play_arrow</i>Speak!</a>
```
Comment (shivamdixit) [inline index.html]: Avoid inline styling

File: index.html [hunk 10]
```diff
@@ -187,24 +185,28 @@ <h2 id="speakerx" style="text-align:center;"><i class="material-icons custom_siz
...
 <a class="waves-effect waves-light btn-large" style="background-color:#2196F3;"><i class="material-icons right">play_arrow</i>Participate</a>
```
Comment (shivamdixit) [inline index.html]: Same – Avoid inline styling

File: index.html [hunk 11]
```diff
@@ -187,24 +185,28 @@ <h2 id="speakerx" style="text-align:center;"><i class="material-icons custom_siz
...
  </div>
 
   </div>
-  <br /><br />
-   <div id="just-a-line"></div>
 </section>
 
 <section id="faqx" style="margin-top : -50px;" class="module content">
+  <br /><br />
```
Comment (shivamdixit) [inline index.html]: Indentation is messed up everywhere?

File: index.html (hunk 8 above)
Comment (sagaragarwal94) [inline index.html]: Removed.

**Pull request merged**